gtda/utils/validation.py

```diff
@@ -154,3 +156,151 @@ def validate_params(parameters, references, exclude=None):
     parameters_ = {key: value for key, value in parameters.items()
                    if key not in exclude_}
     return _validate_params(parameters_, references)
+
+
+def check_list_of_images(X, **kwargs):
+    """Check a list of arrays representing images, by integrating
+    through the input one by one. To pass a test when `kwargs` is ``None``,
+    all images ``x``, ``y`` in `X` must satisfy:
+        - ``x.ndim >= 2``,
+        - ``all(np.isfinite(x))``,
+        - ``x.shape == y.shape``.
+
+    Parameters
+    ----------
+    X : list of ndarray
+        Each entry of `X` corresponds to an image.
+
+    kwargs : dict or None, optional, default: ``None``
```

Comment:
The default is the empty dictionary?

Reply:
(And same for other validation functions in this PR.)

Reply:
I followed your suggestion. is it resolved?